LICENSES/BSD-3-Clause.txt

```diff
@@ -0,0 +1,11 @@
+Copyright (c) <year> <owner>. 
+
```

Comment:
```suggestion
Copyright (C) The DDC development team, see COPYRIGHT.md file

```

Or should it be removed like in the case of the MIT license ?

Reply:
I don't know about removal, that's what we get with `reuse download`, but I wouldn't put the info about the copyright holder here, it's already in each and every file where it belongs. Different files might have different copyright holders.

Reply:
Ok let's keep it that way then